src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Minkowski.cpp

```diff
@@ -103,6 +103,22 @@ tnsr::ii<T, Dim> Minkowski<Dim>::extrinsic_curvature(const tnsr::I<T, Dim> &x,
 noexcept {
   return tnsr::ii<T, Dim>(make_with_value<T>(x, 0.));
 }
+
+template<size_t Dim>
+template<typename T>
+tnsr::Ijj<T, Dim> Minkowski<Dim>::christoffel_2nd_kind(const tnsr::I<T, Dim>& x,
+                                                       const double /*t*/) const
+    noexcept {
+  return tnsr::Ijj<T, Dim>(make_with_value<T>(x, 0.));
+}
+
+template<size_t Dim>
+template<typename T>
+tnsr::iJkk<T, Dim> Minkowski<Dim>::deriv_christoffel_2nd_kind(
```

Comment:
Ditto.  Do we really want all the AnalyticSolutions to have these complicated functions like deriv_christoffel_2nd_kind? I would prefer AnalyticSolutions to have only the simple basic functions (metric and metric derivatives) and not a plethora of quantities derived from them.

Reply:
I removed the code and test for Minkowski to compute christoffel_2nd_kind and its derivative.